### PR TITLE
[CHK-711] - post carts

### PR DESCRIPTION
**Pull request by pietro-tota**
#### List of Changes

- Implemented post carts method
- aligned openapi specifications:
    -  removed 502 error code from post carts response codes
    - aligned ProblemJson to the apim specifications
    - refactored CartRequest.returnrurls.returnErrorOk to returnErrorUrl

#### Motivation and Context
This change is required in order to implements post carts endpoint

#### How Has This Been Tested?
Test have been performed throw junit test and postman

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,205 +1,208 @@
...
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.openapitools</groupId>
+                <artifactId>openapi-generator-maven-plugin</artifactId>
+                <version>5.4.0</version>
+                <executions>
+                    <execution>
+                        <id>session-api</id>
```
Comment (infantesimone) [inline pom.xml:154]: typo of old PR :-)

File: src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/CartService.kt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import org.springframework.stereotype.Service
+import java.net.URI
+
+@Service
+class CartService {
+
+    /*
+     * Logger instance
+     */
+    var logger: Logger = LoggerFactory.getLogger(this.javaClass)
```
Comment (infantesimone) [inline src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/CartService.kt:20]: ditto

**Review by infantesimone — APPROVED** [review]
just some comments, but sounds good!

File: src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/CartService.kt (hunk 2 above)
Comment (pietro-tota) [reply src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/CartService.kt:20]: same as above

**Pull request merged**